packages/shared/forks/ReactFeatureFlags.native-fb.js

```diff
@@ -86,5 +86,7 @@ export const enableHostSingletons = true;
 export const useModernStrictMode = false;
 export const enableFizzExternalRuntime = false;
 
+export const enableFizzIntoContainer = false;
```

Comment:
No reason to leave it off in React Native, testing etc. since there's no renderer being used anyway but it ensures that we don't get blocked trying to remove the flag later and people don't know if it's safe to remove.